application/frontend/src/components/DocumentNode/DocumentNode.tsx

```diff
@@ -61,54 +78,54 @@ export const DocumentNode: FunctionComponent<DocumentNode> = ({ node, linkType,
   }
 
   const isNestedInRelated = (): Boolean => {
-    return hasLinktypeRelatedParent || ( linkType === TYPE_RELATED );
+    return hasLinktypeRelatedParent || (linkType === TYPE_RELATED);
   }
 
   const getTopicsToDisplayOrderdByLinkType = () => {
     return Object.entries(linksByType)
-      .filter( ([type, _]) => !linkTypesExcludedInNesting.includes(type))
-      .filter( ([type, _]) => isNestedInRelated() ? !linkTypesExcludedWhenNestingRelatedTo.includes(type) : true)
+      .filter(([type, _]) => !linkTypesExcludedInNesting.includes(type))
+      .filter(([type, _]) => isNestedInRelated() ? !linkTypesExcludedWhenNestingRelatedTo.includes(type) : true)
   }
 
   const Hyperlink = (hyperlink) => {
-    if ( !hyperlink.hyperlink ) {
+    if (!hyperlink.hyperlink) {
       return <></>;
     }
 
     return <>
-        <span>
-          Reference: 
-        </span>
-        <a href={hyperlink.hyperlink} target="_blank"> {hyperlink.hyperlink}</a>
-      </>
-    
+      <span>
+        Reference:
+      </span>
+      <a href={hyperlink.hyperlink} target="_blank"> {hyperlink.hyperlink}</a>
+    </>
   }
-  
   const SimpleView = () => {
```

Comment:
if you have a component that has just a return, you can use some syntactic goodness, and make it shorter:

```
const SimpleView = () => {
  // no extra logic here
  return (<div></div>);
}

```
bit cleaner:


```
const SimpleView = () => (<div></div>);

```


This is common practice, usually when you can do this, you do.